code/modules/cm_marines/dropship_ammo.dm

```diff
@@ -443,3 +446,37 @@
 		to_chat(user, SPAN_WARNING("The selected drop site is a sheer wall!"))
 		return FALSE
 	return TRUE
+
+
+/obj/structure/ship_ammo/minirocket/smoke
+	name = "\improper 1FW-AH Smoke shell 'Fog'"
+	desc = "The Type 1FW-AH Smoke shell 'Fog' though it lacks a targeting system, its high ammo count and low cost of production means it is a effective way to block enemy visuals, each individual pod contains a chemical mix that produces thick, harmless smoke, or "fog" on impact. Can be loaded into the LAU-229 Rocket Pod."
```

Comment:
```suggestion
	desc = "The Type 1FW-AH Smoke shell 'Fog' though it lacks a targeting system, its high ammo count and low cost of production means it is a effective way to block enemy visuals, each individual pod contains a chemical mix that produces thick, harmless smoke, or 'fog' on impact. Can be loaded into the LAU-229 Rocket Pod."
```

Reply:
Thanks for catching that, ill change it now.

Reply:
Handled.

> Thanks for catching that, ill change it now.